ecs-cli/modules/utils/utils.go

```diff
@@ -58,3 +61,20 @@ func EntityAlreadyExists(err error) bool {
 	}
 	return false
 }
+
+// GetTags parses AWS Resource tags from the flag value
+// users specify tags in this format: key1=value1,key2=value2,key3=value3
+func GetTags(flagValue string, tags []*ecs.Tag) ([]*ecs.Tag, error) {
```

Comment:
This commit is the same as the one in #709. Once one of them is merged, I'll rebase the other on top of it. This code is used by all the tagging functionality in all commands.